96.md

```diff
@@ -275,6 +274,35 @@ The successful response is a 200 OK one with just basic JSON fields:
 }
 ```
 
+## Listing files
+
+`GET $api_url`
+
+**AUTH required**
+
+Returns a list of files linked to the authenticated users pubkey.
+
+Example Response:
+```js
+[
+  { 
+    "id": "<sha256-hash>",
+    "nip94_event": {...},
+    "expires": 1715691139, // unix timestamp
+    "size": 123456, 
+    "alt": "a meme that makes you laugh",
+    "caption": "haha funny meme"
```

Comment:
I admit placing metadata at `nip94_event.tags` wasn't the brightest idea. At the time I just wanted to make sure servers used the same metadata field names and values from NIP-94.

If you want to keep same structure from upload response (without deprecating nip94_event key and introducing an alternative structure for metadata), I think it should be like below: 

```suggestion
    "nip94_event": {
      "tags": [
        ["size", "123456"],
        ["alt", "a meme that makes you laugh"],
        ["expiration",  "1715691139"]
        // ...other metadata
      ]
      "content": "haha funny meme" // caption
    }
```